engines/scumm/verbs.cpp

```diff
@@ -1084,7 +1084,7 @@ void ScummEngine::drawVerb(int verb, int mode) {
 		restoreVerbBG(verb);
 
 		_string[4].charset = vs->charset_nr;
-		_string[4].xpos = vs->curRect.left;
+		_string[4].xpos = vs->origLeft;
```

Comment:
Will this work in all languages, or should there be a check for Hebrew here?

Reply:
I think it should be ok with other languages. @BLooperZ?

Reply:
Added a condition. If you don't have other rejections, please merge it. We can polish it later.

Reply:
I reverted the condition, because I think it can break things. I assign `vs->curRect.left` below (that's the actual fix of the bug), and here we need the original left value.

Reply:
Great